### ISSUE TITLE
Incorrect Watchers Count

### ISSUE THREAD

**Issue by tmobaird** [tmobaird/i-want-to-contribute#95]
![image](https://user-images.githubusercontent.com/7366444/34651900-059abffa-f3a5-11e7-9308-64d39f461cbb.png)

The total highlighted in the image above says 9 watchers, although GitHub web reports the total number of watchers to be 4.

I haven't read the GitHub API docs deeply on this, but I assume it is something similar to the Issue/Pull Request total.

**Comment by kbeaulieu802** [comment]
This is due to a change GitHub made in 2012, the watcher endpoints provide starring data, to get the correct count of watchers this should be updated to use the subscription endpoints or more specifically subscribers_count.

**Comment by cubadomingo** [comment]
Good catch, would you like to take on this issue?

**Comment by tmobaird** [comment]
I'm not gonna lie, I'm a little bummed that the Github API repo search query we use doesn't include the subscribers_count in it's results.

Does anyone think it's worth it to add another Github query for this little bit of information?

_PS. And the argument for using V4 GraphQL api gets even stronger..._

**Comment by kbeaulieu802** [comment]
I don't see any issues with making this change even if it is for just a little bit of info. Like you said @tmobaird this definitely adds to the argument for switching to V4 and GraphQL, which i can take a look at. Its a good excuse to learn GraphQL.

**Comment by kbeaulieu802** [comment]
@tmobaird i spent sometime looking into what it would take to implement this and it looks like it would require more work than i initially thought. To get the subscibers count we could use the get repositories endpoint but it would require the passing of the repo owner, I think this is something that should be postponed until v4 of the github api has been implemented.

**Comment by tmobaird** [comment]
@kbeaulieu802 I totally agree. If you're looking to do some stuff with the v4 API feel free to start doing some work maybe converting an endpoint or two to using the graphQL github api!

For some context, [server/routes.js](https://github.com/tmobaird/i-want-to-contribute/blob/master/src/server/routes.js) includes all of the actual endpoints that hit the Github API.

**Comment by kbeaulieu802** [comment]
@tmobaird Sounds good!
